src/bun.js/api/server.zig

```diff
@@ -658,6 +658,8 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
         request_body_content_len: usize = 0,
         sink: ?*ResponseStream.JSSink = null,
         byte_stream: ?*JSC.WebCore.ByteStream = null,
+        request_body_waiting_for_data: ?*JSC.WebCore.Body.Value = null,
```

Comment:
What ensures this `Body.Value` pointer is alive for the duration of the request?

Reply:
it is alive when .json() is triggered, and waiting for request body data. The purpose of this is to avoid using `request_js_object`'s body value dirrectly, which would make its internal status not right sometimes.